lesson2/MainActivity.java/lib/src/main/java/com/example/Fraction.java

```diff
@@ -35,14 +35,12 @@ public Fraction simplify() {
         int numGCD = gcd(numerator, denominator);
         numerator = numerator/numGCD;
         denominator = denominator/numGCD;
-        Fraction simpleFrac= new Fraction(numerator,denominator);
-        return simpleFrac;
-
+        return new Fraction(numerator, denominator);
```

Comment:
Instead of saving your new Fraction to a variable `simpleFrac` and then returning it, you can simply return your new Fraction immediately on one line!  Take up one line instead of two :smile: